### PR TITLE
Fix use of path-or-url

### LINKED ISSUE #62: [Question] Worker and domains
Hi. This is more a question than a real bug report:

I'd like to use the _IndexedDb_ from within the worker script (I'm using the pool on the non-worker side) and share that _IndexedDb_ with the main application.  Sadly that's not possible, because the worker gets injected by a Blob-URL and - as a result - runs in a different security domain. There's no shareable _IndexedDb_ possible with that setup - except when commenting some lines to use the IE fallback.

Any idea how that can be achieved?

### PR DESCRIPTION
**Pull request by cajus**
This mini commit makes the code example reflect the code (`config.browser` vs `config.web`) and takes care about the _path_ part of _path-or-url_.

Problem is (at least with a current chrome), that the script is not loaded when using an absolute path. It seems to require a full URL.

closes #62

### CODE REVIEW

**Comment by cajus** [comment]
No idea why 'pool-functional' fails in the moment. Doesn't look related on the first glance.

**Comment by andywer** [comment]
Thanks for the PR!

Will have a look at the failing tests.

**Comment by andywer** [comment]
Sorry for the long delay, but I just managed to fix the build (#67)!

**Comment by andywer** [comment]
Just thinking that a regression test might be nice... Do you have any spare time to add a test case that fails without your fix?

File: src/worker.browser/worker.js [hunk 1]
```diff
@@ -18,8 +18,12 @@ function joinPaths (path1, path2) {
 }
 
 function prependScriptUrl(scriptUrl) {
-  const prefix = getConfig().basepath.web;
-  return prefix ? joinPaths(prefix, scriptUrl) : scriptUrl;
+  let basePath = getConfig().basepath.web;
+  if (basePath && !basePath.match(/^\w+:\/\//)) {
+    // Turn path into full URL, since referencing a `http://` path from a `blob://` URL won't work
+    basePath = `${document.location.protocol}//${document.location.host}${basePath}`;
```
Comment (andywer) [inline src/worker.browser/worker.js]: Need to check: Does `document.location.protocol` equal `http:`|`https:`, or is it `blob:`?

**Comment by andywer** [comment]
Closing due to inactivity.

**Pull request closed without merging**